translations/pt_BR/i18n.ts

```diff
@@ -637,7 +637,7 @@ var t_pt_BR: TalkyardTranslations = t = {
     ClickUnaccept: "Clique para desafazer a aceitação desta resposta",
     PostAccepted: "Este post foi aceitado como resposta",
 
-    NumLikes: (num: number) => num === 1 ? "1 Curte" : num + " Curtem",
+    NumLikes: (num: number) => num === 1 ? "1 Curtiu" : num + " Curtiram",
```

Comment:
Are bot `Curte` and `Curtiu` Brazilian Portuguese?  (none of them Portuguese Portuguese?)

I'm curious about what's the difference :- )

Reply:
Yeah, both are correct in brazilian portuguese.
Curte (in the present) is also correct, but "Curtiu" (in the past) is the most used form on social networks, for instance in facebook:
![_2__Facebook](https://user-images.githubusercontent.com/847070/95909454-e6886d00-0d74-11eb-9da2-29c5932e9bc8.png)